File: JavaApp/GetPhoto/app/src/main/java/com/example/getphoto/MainActivity.java
package com.example.getphoto;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    ImageView iv_main;
    
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: KotlinApp/RecyclerviewPractice/app/src/main/java/com/example/recyclerviewpractice/MainActivity.kt
package com.example.recyclerviewpractice

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

class MainActivity : AppCompatActivity() {

    var studentList = arrayListOf<Student>(
        Student(1315, "임세현", "android"),
        Student(1315, "임세현", "android"),
        Student(1315, "임세현", "android")
    )

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val rvStudent: RecyclerView = findViewById(R.id.rvStudent)

        val mAdapter = MainRvAdapter(this, studentList)
        rvStudent.adapter = mAdapter

        val lm = LinearLayoutManager(this)
        rvStudent.layoutManager = lm
        rvStudent.setHasFixedSize(true)
    }
}